src/signer_node/message_processor/process_blockvss.rs

```diff
@@ -31,103 +31,122 @@ where
     T: TapyrusApi,
     C: ConnectionManager,
 {
+    let new_shared_block_secrets = match store_received_vss(
```

Comment:
I think that changing a return type of process_blockvss to Result<NodeState, Error> may make code simpler because it should return prev_state.clone() when some errors have occurred or prev_state is not master/member.

Reply:
Can I keep this changes until after my current task ( #29 ) finished? It should be affect a lot for my local changes.

Actually, I got such a thought too and i tried it. but i stoped it because it requires to change for all message process function and its unit test.